app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -186,7 +186,14 @@ def vm_migrate(vm, options = {})
   end
 
   def unsupported_migration_options
-    [:storage, :respool, :folder, :datacenter, :host_filter]
+    [:storage, :respool, :folder, :datacenter, :host_filter, :cluster]
+  end
+
+  # Migrations are supposed to work only in one cluster. If more VMs are going
+  # to be migrated, all have to live on the same cluster, otherwise they can
+  # not be migrated together.
+  def supports_migrate_for_all?(vms)
```

Comment:
@gmcculloug from the `application_controller.rb` in https://github.com/ManageIQ/manageiq-ui-classic/pull/207